code/scripting/api/objs/model.cpp

```diff
@@ -829,38 +799,27 @@ ADE_FUNC(__len, l_Thrusterbank, NULL, "Number of thrusters on this thrusterbank"
 
 ADE_INDEXER(l_Thrusterbank, "number Index", "Array of glowpoint", "glowpoint", "Glowpoint, or invalid glowpoint handle on failure")
 {
-	thrusterbank_h *tbh = NULL;
-	const char* s       = nullptr;
-	glowpoint_h *glh = NULL;
-
-	if (!ade_get_args(L, "os|o", l_Thrusterbank.GetPtr(&tbh), &s, l_Glowpoint.GetPtr(&glh)))
-		return ade_set_error(L, "o", l_Glowpoint.Set(glowpoint_h()));
+	thrusterbank_h *tbh = nullptr;
+	int idx = -1;
 
-	if (!tbh->isValid() || s==NULL)
+	if (!ade_get_args(L, "oi", l_Thrusterbank.GetPtr(&tbh), &idx))
```

Comment:
same here, does this need to keep compat?

Reply:
Same response - this was a numeric indexer before, and it's still a numeric indexer.